src/store/modules/purchaseOrder/actions.ts

```diff
@@ -0,0 +1,46 @@
+import { PurchaseOrdersService } from "@/services/PurchaseOrderService";
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import PurchaseOrderState from './PurchaseOrderState'
+import * as types from './mutation-types'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+import emitter from '@/event-bus'
+
+
+const actions: ActionTree<PurchaseOrderState, RootState> = {
+
+  // Find Product
+  async findPurchaseOrders ({ commit, state }, payload) {
+
+    // Show loader only when new query and not the infinite scroll
+    // if (payload.viewIndex === 0) emitter.emit("presentLoader");
+
+    let resp;
+
+    try {
+      resp = await PurchaseOrdersService.fetchPurchaseOrders(payload)
+
+      // resp.data.response.numFound tells the number of items in the response
+      if (resp.status === 200 && !hasError(resp) && resp.data.grouped) {
+        const orders = resp.data.grouped.orderId.groups
+        commit(types.PURCHASE_ORDER_UPDATED, {
+          list: orders,
+          total: orders.length
```

Comment:
The total value here should be the no of purchase orders, but here we are storing the length of orders received.

Reply:
Changed this value from length to ngroups which comes in API response.